src/main/java/filter/expression/Qualifier.java

```diff
@@ -93,6 +92,47 @@ public Qualifier(String name, List<SortKey> keys) {
         this.sortKeys = new ArrayList<>(keys);
     }
 
+    public static FilterExpression replaceMilestoneAliases(IModel model, FilterExpression expr) {
+        List<Qualifier> metaQualifiers = expr.find(Qualifier::isMetaQualifier);
+        List<String> repoIds = getReposInMetaQualifiers(metaQualifiers);
```

Comment:
Use `getContentOfMetaQualifier(expr, REPO)` here? Can also rename it to `getMetaQualifierContent`.